neptune/new/run.py

```diff
@@ -175,3 +180,218 @@ def _startup(self, debug_mode):
+
+    def pop(self, path: str, wait: bool = False) -> None:
+        """Removes the field stored under the path completely and all data associated with it.
+
+        Args:
+            path (str): Path of the field to be removed.
+            wait (bool, optional): If `True` the client will first wait to send all tracked metadata to the server.
+                This makes the call synchronous. Defaults to `True`.
+
+        Examples:
+            >>> import neptune.new as neptune
+            >>> run = neptune.init()
+
+            >>> run['parameters/learninggg_rata'] = 0.3
+
+            >>> # Delete a field along with it's data
+            ... run.pop('parameters/learninggg_rata')
+
+            >>> run['parameters/learning_rate'] = 0.3
+
+            >>> # Training finished
+            ... run['trained_model'].upload('model.pt')
+            >>> # 'model_checkpoint' is a File field
+            ... run.pop('model_checkpoint')
+
+        You may also want to check `pop docs page`_.
+
+        .. _pop docs page:
+           https://docs.neptune.ai/api-reference/run#.pop
+        """
+        AttributeContainer.pop(self, path=path, wait=wait)
+
+    def wait(self, disk_only=False) -> None:
+        """Wait for all the tracking calls to finish.
+
+        Args:
+            disk_only (bool, optional, default is False): If `True` the process will only wait for data to be saved
+                locally from memory, but will not wait for them to reach Neptune servers.
+                Defaults to `False`.
+
+        You may also want to check `wait docs page`_.
+
+        .. _wait docs page:
+            https://docs.neptune.ai/api-reference/run#.wait
+        """
+        AttributeContainer.wait(self, disk_only=disk_only)
+
+    def sync(self, wait: bool = True) -> None:
+        """Synchronizes local representation of the run with Neptune servers.
+
+        Args:
+            wait (bool, optional, default is True): If `True` the process will only wait for data to be saved
+                locally from memory, but will not wait for them to reach Neptune servers.
+                Defaults to `True`.
+
+        Examples:
+            >>> import neptune.new as neptune
+
+            >>> # Connect to a run from Worker #3
+            ... worker_id = 3
+            >>> run = neptune.init(run='DIST-43', monitoring_namespace='monitoring/{}'.format(worker_id))
+
+            >>> # Try to access logs that were created in meantime by Worker #2
+            ... worker_2_status = run['status/2'].fetch() # Error if this field was created after this script starts
+
+            >>> run.sync() # Synchronizes local representation with Neptune servers.
+            >>> worker_2_status = run['status/2'].fetch() # No error
+
+        You may also want to check `sync docs page`_.
+
+        .. _sync docs page:
+            https://docs.neptune.ai/api-reference/run#.sync
+        """
+        AttributeContainer.sync(self, wait=wait)
```

Comment:
what do you think about doing `return AttributeContainer....` everywhere (i.e. `return` always, even if it's `None`)? wouldn't hurt, I guess?

Reply:
Sure